rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -560,26 +556,25 @@ while (assetId = stake_pools.has_next) is (yes)
         :reward.total_rewards += new_reward;
         note left
             In case not having enough funds
-            this would fail. But it would recover
-            once the funds are available.
+            the pool will pause. it will recover
+            once enough rewards are accumulated.
```

Comment:
the pool won't "recover", it will resume once there are enough funds available to accumulate again